tests/codegen/call_path.cpp

```diff
@@ -0,0 +1,14 @@
+#include "common.h"
+
+namespace bpftrace {
+namespace test {
+namespace codegen {
+
+TEST(codegen, call_path)
+{
+  test("kfunc:filp_close { path(args->filp->f_path); }", NAME);
```

Comment:
If this codegen test doesn't already exist, want to add one with the optional size?